core/src/sigverify_stage.rs

```diff
@@ -229,18 +227,19 @@ impl SigVerifyStage {
             num_packets,
         );
 
-        let mut dedup_time = Measure::start("sigverify_dedup_time");
-        let dedup_fail = dedup_packets(bloom, &mut batches) as usize;
-        dedup_time.stop();
-        let valid_packets = num_packets.saturating_sub(dedup_fail);
-
+        //50ns per packet with a single core
         let mut discard_time = Measure::start("sigverify_discard_time");
-        if valid_packets > MAX_SIGVERIFY_BATCH {
+        if num_packets > MAX_SIGVERIFY_BATCH {
             Self::discard_excess_packets(&mut batches, MAX_SIGVERIFY_BATCH)
```

Comment:
If we discard before dedupe, then we will end up with less than `MAX_SIGVERIFY_BATCH` per batch. Is that desirable? Can we do a hybrid where we have `MAX_DEDUP_BATCH` before dedupe and then `MAX_SIGVERIFY_BATCH` before sigverify?

Reply:
MAX_SIGVERIFY_BATCH does a discard per IP right?  I would imagine dedup will have few hits if all the IPs are different.

Reply:
Yeah, if someone sends dupes from multiple IPs they can eat away at throughput though right?

Reply:
We should be doing qos by stake weight there.  @sakridge is working on it.